opencensus/trace_exemplar.go

```diff
@@ -19,15 +19,25 @@ package opencensus
 
 // [START monitoring_opencensus_configure_trace_exemplar]
 import (
+	"fmt"
 	"time"
 
 	googlepb "github.com/golang/protobuf/ptypes/timestamp"
 	distributionpb "google.golang.org/genproto/googleapis/api/distribution"
 	monitoringpb "google.golang.org/genproto/googleapis/monitoring/v3"
+	"google.golang.org/protobuf/types/known/anypb"
 )
 
-func createDataPointWithExemplar() *monitoringpb.Point {
+func createDataPointWithExemplar(projectID string) *monitoringpb.Point {
```

Comment:
Please add a comment to this function: https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#comment-functions-and-packages

Reply:
Please supply example value for projectID: https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#function-arguments-for-snippets

Reply:
Fixed.